src/arcgateway/azext_arcgateway/commands.py

```diff
@@ -0,0 +1,15 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+# from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):  # pylint: disable=unused-argument
+    self.command_table["arcgateway settings update"] = Update(loader=self)
```

Comment:
import `Update`?

Reply:
I was trying to customize 'az arcgateway settings update'. Do you know how to overwrite the command here?

Reply:
updated custom and commands files to correctly overwrite the update command.